internal/pkg/service/buffer/storage/local/writer/base/base.go

```diff
@@ -29,21 +24,14 @@ type Writer struct {
 }
 
 func NewWriter(logger log.Logger, clock clock.Clock, slice *storage.Slice, dirPath string, filePath string, chain *writechain.Chain) *Writer {
-	w := &Writer{
+	return &Writer{
 		chain:    chain,
 		syncer:   disksync.NewSyncer(logger, clock, slice.LocalStorage.Sync, chain),
 		logger:   logger,
 		slice:    slice,
 		dirPath:  dirPath,
 		filePath: filePath,
 	}
-
-	// Add a small buffer before the file
-	w.chain.PrependWriter(func(writer writechain.Writer) io.Writer {
-		return bufio.NewWriterSize(writer, int(fileBufferSize.Bytes()))
-	})
-
-	return w
```

Comment:
Tento buffer sa vytvara uz iba ked nie je pouzita kompresia, kedze "compression writers" maju interne buffers.

Na rychlost to nemá meratelny vplyv ak je pouzita kompresia, je to rýchlejsie iba v situáciach bez kompressie (o `10-20%`). Teda pri pouzitej kompresii sa usetri nejaka memory, ktora by bola alokovana zbytocne.